web/pgadmin/tools/sqleditor/templates/sqleditor/sql/default/sequences.sql

```diff
@@ -0,0 +1,10 @@
+SELECT DISTINCT att.attname as name, att.attnum as OID, seq.seqtypid
```

Comment:
This can be clubbed with get_columns_types. I don't think we need a separate SQL for this.

Reply:
BTW, why didn't we use `pg_get_serial_sequence` to check if it is a serial column?

Reply:
Had discussion with Khushboo for this. If we use that function we will have to call it for all the columns, but using the query we get all the details at once.

Reply:
If you use that function in the nodes.sql query of the column by just adding `pg_get_serial_sequence`, you will get for all the columns at once right?

Reply:
As per the documentation `pg_get_serial_sequence ( table text, column text )` we will be requiring to pass the table name and column name in text format.